models/hashtag_test.go

```diff
@@ -1,20 +1,167 @@
 package models
 
 import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
 	"testing"
-
-	"github.com/stretchr/testify/assert"
+	"github.com/go-xorm/core"
+	"github.com/go-xorm/xorm"
+	_ "github.com/mattn/go-sqlite3"
+	. "github.com/smartystreets/goconvey/convey"
```

Comment:
Somehow convey got changed to assert. All I did was rebase. Recoding this is out of the scope of this sprint.